packages/govuk-frontend/src/govuk/components/tag/_index.scss

```diff
@@ -4,6 +4,12 @@
 
     display: inline-block;
 
+    // set a max-width along with overflow-wrap: break-word below for instances
+    // where a tag has a single long word and could overflow its boundaries.
+    // The max-width is necessary as break-word requires a bounding box to base
+    // where to break off of.
+    max-width: 160px;
```

Comment:
If this comes from `$govuk-breakpoints` "mobile" (320px ÷ 2) should we add the calculation?

Reply:
It doesn't but that's a neat coincidence. Maybe valuable to make it dynamic based on the breakpoints map.